packages/skin/src/routes/_index/component/chip/+page.marko

```diff
@@ -26,10 +26,366 @@
         </span>
     </highlight-code>
 
-    <h3 id="interactive-chip">
-        Interactive Chip
+    <h3 id="toggle-chip">
+        Toggle Chip
     </h3>
-    <p>An interactive chip has a delete button inside for chip removal.</p>
+    <p>An toggle chip toggles between two states, selected and unselected.</p>
```

Comment:
selected and unselected are for th `aria-selected` propery. Let's make sure all references are updated and correct for aria-pressed.